### DISCUSSION TITLE
Contribution templates

### DISCUSSION THREAD

**Discussion by delisma** [gc-da11yn/gc-da11yn.github.io#197] [General (Open Discussions)]
@shawnthompson Have you seen this [type of bug report template](https://github.com/vitejs/vite/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml)? It's a fillable form in YAML!!!
I'll talk to @nschonni about how to leverage them to the max.

**Comment by nschonni** [comment]
You can see an example of what the MDN folks are looking at for i18n versions when they have those "Report on GitHub" links on the localized pages https://github.com/mdn/translated-content/pull/9073

**Comment by shawnthompson** [comment]
I like this we should add it to a project and implement it. Thanks!
